apistar/templates/docs/layout/content.html

```diff
@@ -1,9 +1,7 @@
 <div class="row intro">
 <div class="col-md-6 intro-title">
     <h1>{{ document.title|default('API Star', True) }}</h1>
-    {% if document.description %}
-    <p>{{ document.description }}</p>
-    {% endif %}
+    <p>{{ document.description|default('description', True) }}</p>
```

Comment:
Probably wrap `{% if document.description %}` around this `<p>` block - we don't want to provide a default text string here.